raiden-dapp/src/types.d.ts

```diff
@@ -7,6 +7,11 @@ export type Tokens = { [token: string]: Token };
 export type Transfers = { [secretHash: string]: RaidenTransfer };
 export type ChannelAction = 'close' | 'deposit' | 'settle';
 
+export interface ConnectOptions {
+  uploadedState: string;
+  subkey?: true | undefined;
```

Comment:
`| undefined` here is redundant the `?` implies that it is optional and might have an undefined value